config/esp32/components/chip/Kconfig

```diff
@@ -311,7 +311,7 @@ menu "CHIP Device Layer"
                 Enables or Disables ICD server
 
         config ICD_SLOW_POLL_INTERVAL
-            int "ICD Slow Polling Interval"
+            int "ICD Slow Polling Interval(ms)"
```

Comment:
the config should have units in it since it ends up in a define and that makes code more readable long term.

Could we change this to `ICD_SLOW_POLL_INTERVAL_MS` and `ICD_FAST_POLL_INTERVAL_MS` and similar?

Reply:
Added the unit suffixes